core/src/main/java/nu/yona/server/CoreConfiguration.java

```diff
@@ -121,6 +122,7 @@ public JavaMailSender javaMailSender()
 	private BeanFactory beanFactory;
 
 	@Bean
+	@Primary
```

Comment:
I didn't fix it. I don't think it matters much, as all end points are anyway documented. I expect we'll directly use an endpoint.